### PR TITLE
TW-380 Document GraphQL schema introspection

### PR DESCRIPTION
**Pull request by jkonrath-postman**

### CODE REVIEW

File: src/pages/docs/sending-requests/supported-api-frameworks/graphql.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ There are a few ways for you to author and send a GraphQL query using Postman:
 
 ## Using Postman's built-in support for GraphQL
 
-Postman has built-in support for [sending GraphQL queries in the request body](#sending-graphql-queries-in-the-request-body), [using GraphQL variables](#using-graphql-variables), [importing GraphQL schemas](#importing-graphql-schemas), and [enabling GraphQL query autocompletion](#enabling-graphql-query-autocompletion).
+Postman has built-in support for [sending GraphQL queries in the request body](#sending-graphql-queries-in-the-request-body), [using GraphQL variables](#using-graphql-variables), and [importing GraphQL schemas](#importing-graphql-schemas).
```
Comment (giridharvc7) [inline src/pages/docs/sending-requests/supported-api-frameworks/graphql.md:56]: > Importing GraphQL Schemas 

We can change this to `Introspection and Importing GraphQL Schemas`

File: src/pages/docs/sending-requests/supported-api-frameworks/graphql.md [hunk 2]
```diff
@@ -62,20 +62,24 @@ Postman has built-in support for [sending GraphQL queries in the request body](#
...
 
-![Using GraphQL variables](https://assets.postman.com/postman-docs/graphql-variables.jpg)
+![Using GraphQL variables](https://assets.postman.com/postman-docs/graphql-variables-v9-14.jpg)
 
 > You can also [use Postman variables](/docs/sending-requests/variables/) as data inputs for GraphQL variables using `{{variable}}` syntax.
 
 ### Importing GraphQL schemas
 
-To enable autocompletion, you need to add a GraphQL schema. To [create or import a GraphQL schema](/docs/designing-and-developing-your-api/creating-an-api/) into Postman:
+To enable autocompletion, your request needs a GraphQL schema. Postman will automatically fetch a schema based on the request URL, using introspection. A message is displayed next to the schema selection list when Postman is able to auto-fetch a schema. If your schema has changed, select <img alt="Refresh icon" src="https://assets.postman.com/postman-docs/icon-refresh-v9-5.jpg" width="14px" style="vertical-align:middle;margin-bottom:5px"> to refresh it.
```
Comment (giridharvc7) [inline src/pages/docs/sending-requests/supported-api-frameworks/graphql.md:78]: using *GraphQL Introspection*

**Review by giridharvc7 — COMMENTED** [review]
Have suggested some changes

File: src/pages/docs/sending-requests/supported-api-frameworks/graphql.md (hunk 2 above)
Comment (HeatherStenson) [inline src/pages/docs/sending-requests/supported-api-frameworks/graphql.md:78]:
```suggestion
To enable autocompletion, your request needs a GraphQL schema. Postman will automatically fetch a schema based on the request URL, using introspection. A message is displayed next to the schema selection list when Postman is able to auto-fetch a schema. If your schema has changed, select the refresh icon <img alt="Refresh icon" src="https://assets.postman.com/postman-docs/icon-refresh-v9-5.jpg" width="14px" style="vertical-align:middle;margin-bottom:5px"> to pull in the changes.
```

**Review by HeatherStenson — APPROVED** [review]

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

**Pull request merged**